code/style.css

```diff
@@ -2,6 +2,376 @@
+  align-self: center;
+}
+
+.hero-banner {
+  background-image: url(./images/ink_on_pink.jpeg);
+  background-attachment: fixed;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+/********* SIGNUP FORM AND FAQ *********/
+
+.info-box {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  padding: 2rem 0;
+}
+
+.sign-up { 
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  background-color: #ff5d38; 
+  width: 30%;
+  padding: 3em;
+  -webkit-appearance: none;
+}
+
+.name {
+  color: white;
+  padding-left: 0.5em;
+  outline: none;
+  background-color: #fcb2a1;
+  line-height: 1.8em;
+  border: 1px;
+  border-color: #ff5d38;
+  border-radius: 40px;
+  margin: 0.6em 0;
+  width: 70%;
+  
+}
+
+.name:focus {
+  background-color: #C0BBF9;
+}
+
+.email {
+  padding-left: 0.5em;
+  color: white;
+  outline: none;
+  background-color: #fcb2a1;
+  line-height: 1.8em;
+  border: 0;
+  border-radius: 40px;
+  margin: 0.6em 0;
+  width: 70%;
+}
+
+.email:focus {
+  background-color: #C0BBF9;
+}
+
+  .password {
+    padding-left: 0.5em;
+    color: white;
+    outline: none;
+    background-color: #fcb2a1;
+    line-height: 1.8em;
+    border: 0;
+    border-radius: 40px;
+    margin: 0.6em 0;
+    width: 70%;
+  }
+
+  .password:focus {
+    background-color: #C0BBF9;
+  }
+
+  ::placeholder {
+    font-family: 'Roboto Mono', monospace;
+    font-size: 1.2em;
+    color: white;
+  }
+
+    /********* RADIO INPUTS *********/
+
+  .radio-inputs {
+    font-size: 0.7em;
+    display: block;
+    position: relative;
+    margin-top: 1em;
+    line-height: 1rem;
+    padding-left: 20px;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+
+  .radio-inputs input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+  }
+
+  .checkmark {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 15px;
+    width: 15px;
+    background-color: #4357ef;
+    border-radius: 50%;
+  }
+
+  .radio-inputs:hover input ~ .checkmark {
+    background-color: #6576f7;
+  }
+
+  .radio-inputs input:checked ~ .checkmark {
+    background-color: #2c3cb7;
+  }
+
+  .checkmark:after {
+    content: "";
+    position: absolute;
+    display: none;
+  }
+
+  .radio-inputs input:checked ~ .checkmark:after {
+    display: block;
+  }
+
+  .radio-inputs .checkmark:after {
+    top: 4.5px;
+    left: 4.5px;
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    background: white;
+ }
+
+
+
+/********* CHECKBOX *********+*/
+
+  .terms {
+    padding: 2em 0 2em 0;
+    font-size: 0.7em;
+    display: block;
+    position: relative;
+    margin-bottom: 12px;
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;  
+  }
+
+
+  /********** I WAS TRYING TO STYLE THE CHECKBOX *********/
```

Comment:
Nice! This code still be saved if you do a commit and then delete it before the pull request. It's great if you give it another try later on!